pybamm/models/li_ion/single_particle_model.py

```diff
@@ -9,91 +9,60 @@
 class SPM(pybamm.BaseModel):
     """Single Particle Model for li-ion.
 
-    Attributes
-    ----------
-
-    rhs: dict
-        A dictionary that maps expressions (variables) to expressions that represent
-        the rhs
-    initial_conditions: dict
-        A dictionary that maps expressions (variables) to expressions that represent
-        the initial conditions
-    boundary_conditions: dict
-        A dictionary that maps expressions (variables) to expressions that represent
-        the boundary conditions
-    variables: dict
-        A dictionary that maps strings to expressions that represent
-        the useful variables
+    **Extends**: :class:`pybamm.BaseModel`
 
     """
 
     def __init__(self):
         super().__init__()
 
         # Variables
-        cn = pybamm.Variable("cn", domain="negative particle")
-        cp = pybamm.Variable("cp", domain="positive particle")
+        c_n = pybamm.Variable("Particle concentration", domain="negative particle")
+        c_p = pybamm.Variable("Particle concentration", domain="positive particle")
```

Comment:
Does it matter that both `c_n` and `c_p` are called `"Particle concentration"`, or should they be called `"Negative particle concentration"` and `"Positive particle concentration"`? Does having the same variable name cause an issue, even if they have different domains?